react/features/overlay/actions.js

```diff
@@ -0,0 +1,40 @@
+import {
+    SUSPEND_DETECTED,
+    MEDIA_PERMISSION_PROMPT_VISIBILITY_CHANGED
+} from './actionTypes';
+import './reducer';
+
+/**
+ * Signals that suspend was detected.
+ *
+ * @returns {{
+ *     type: SUSPEND_DETECTED
+ * }}
+ * @public
+ */
+export function suspendDetected() {
+    return {
+        type: SUSPEND_DETECTED
+    };
+}
+
+/**
+ * Signals that the prompt for media permission is visible or not.
+ *
+ * @param {boolean} isVisible - If the value is true - the prompt for media
+ * permission is visible otherwise the value is false/undefined.
+ * @param {string} browser - The name of the current browser.
+ * @returns {{
+ *     type: MEDIA_PERMISSION_PROMPT_VISIBILITY_CHANGED,
+ *     isVisible: {boolean},
+ *     browser: {string}
```

Comment:
Now or later we'll always need 3 changes. 

I would do it but there are some decisions to be made like:
1) should we export RTCBrowserType from lib-jitsi-meet and start using it from jitsi-meet
2) should we create js util package where we can put RTCBrowserType and use it in lib-jitsi-meet and jitsi-meet.
3) or maybe we should use 3rd party library for browser detection in jitsi-meet
Probably there are a lot other options.

Recently there was discussion about this topic but I don't think we have final decision yet. 

I don't see a point to make this change now if we have to fix it again later. Also the scope of the PR is to refactor the overlays to work with react and that's how they were working before. So I really prefer to fix this later.